File: lesson6/python3/lesson6_1.py
#!/usr/bin/python3
#: lesson6/python3/lesson6_1.py


def sum_digits(numb_n):
    summ = 0
    while numb_n != 0:
        summ += numb_n % 10
        numb_n //= 10
    return summ


print(sum_digits(12345))

File: lesson6/c_c++/lesson6_1.c
/*
 * lesson6/c_c++/lesson6_1.c
 * Функции
 */
#include <stdio.h>

int summDigits(int numb_n) {
    int sum = 0;
    while (numb_n != 0) {
        sum += numb_n % 10;
        numb_n /= 10;
    }
    return sum;
}

int main() {
    printf("%d\n", summDigits(12345));
    return 0;
}
